routes/calendar.routes.js

```diff
@@ -21,35 +21,110 @@ router.get("/", isAuthenticated, async (req, res, next) => {
       latestVolumes.push(foundVolume);
     }
 
+    if (latestVolumes.length === 0) {
+      res.sendStatus(404);
+      return;
+    }
+    latestVolumes.sort((a, b) => a.releaseDate - b.releaseDate);
+    res.status(200).json(latestVolumes);
+  } catch (err) {
+    next(err);
+  }
+});
+
+// Read Month Releases
+router.get("/:year/:month", isAuthenticated, async (req, res, next) => {
+  try {
+    const { _id } = req.user;
+    let latestVolumes = [];
+    const { year, month } = req.params;
+
+    const foundFavorites = await Favorite.find({ user: _id });
+    for (let i = 0; i < foundFavorites.length; i++) {
+      const { series } = foundFavorites[i];
+      if (year && month) {
+        const foundVolume = await MangaVolume.findOne({
+          $and: [
+            { series: series },
+            {
+              releaseDate: {
+                $gte: new Date(`${year}/${month}/01`),
+                $lte: new Date(`${year}/${month}/31`),
```

Comment:
This "31st" approach has issues because March 1st will be included in February. Instead, I think you could do `$lt: startOfNextMonth` with `const startOfNextMonth = new Date(year, month, 1)`.

Reply:
Try `new Date("2022/02/31")`. For me in node, it gives me: `2022-03-02T23:00:00.000Z`.